### PR TITLE
Apply clang format

### PR DESCRIPTION
**Pull request by hassount**
# Description
Apply clang format on files that would not impact current dev projects

# Issue

# Author(s)
@hassount 

# Performance impact

- [ ] quality
- [ ] memory
- [ ] speed
- [ ] 8 bit
- [ ] 10 bit
- [x] N/A

# Test set
- [ ] obj-1-fast can be found [here](https://media.xiph.org/video/derf/objective-1-fast.tar.gz)
- [ ] other
- [x] N/A

# Merge method
- [x] Allow the maintainer to squash and merge when PR is ready to create a 1-commit to the master branch. The maintainer will be able to fix typos / combine commit messages to create a more readable 1-commit message or use whatever is stated in the 'Description' section
- [ ] I will clean up my commits and the maintainer shall use 'rebase and merge' to the master branch

### CODE REVIEW

**Comment by 1480c1** [comment]
After we completely run clang-format, should I try to add a job that uses https://github.com/llvm/llvm-project/blob/master/clang/tools/clang-format/clang-format-diff.py to run something like `git diff -U0 --no-color FETCH_HEAD | python3 <(curl -s https://raw.githubusercontent.com/llvm/llvm-project/master/clang/tools/clang-format/clang-format-diff.py) -p1` to show any needed changes that can also be applied automatically by piping the output to `git apply`

**Comment by hassount** [comment]
> After we completely run clang-format, should I try to add a job that uses https://github.com/llvm/llvm-project/blob/master/clang/tools/clang-format/clang-format-diff.py to run something like `git diff -U0 --no-color FETCH_HEAD | python3 <(curl -s https://raw.githubusercontent.com/llvm/llvm-project/master/clang/tools/clang-format/clang-format-diff.py) -p1` to show any needed changes that can also be applied automatically by piping the output to `git apply`

sure, that would be great :)

File: Source/Lib/Encoder/Codec/EbInitialRateControlProcess.c [hunk 1]
```diff
@@ -189,17 +194,17 @@ InitialRateControlReorderEntry *determine_picture_offset_in_queue(
...
-                  encode_context_ptr
-                      ->initial_rate_control_reorder_queue
-                          [encode_context_ptr->initial_rate_control_reorder_queue_head_index]
-                      ->picture_number);
+    queue_entry_index = (int32_t)(
+        pcs_ptr->picture_number -
+        encode_context_ptr
+            ->initial_rate_control_reorder_queue
+                [encode_context_ptr->initial_rate_control_reorder_queue_head_index]
+            ->picture_number);
```
Comment (chkngit) [inline Source/Lib/Encoder/Codec/EbInitialRateControlProcess.c:202]: What I don't like is why the tool splits encode_context_ptr ->initial_rate_control_reorder_queue  into 2 lines!!
makes reading the code a bit difficult

Comment (hassount) [reply Source/Lib/Encoder/Codec/EbInitialRateControlProcess.c:202]: I agree, but it is due to the fact that our variable names are too long :/ so the tool needs to split it to be able to fit within the line limit. Infact some of the files such as EbEncHandle.c are untouchable 

we would need to make an effort to reduce the length of our variable names so we avoid having scenarios such as the above.

@chkngit would you like me to revert back this file for now ?

Comment (chkngit) [reply Source/Lib/Encoder/Codec/EbInitialRateControlProcess.c:202]: how come? even if we keep that variable not split, I see room in the first line to fit both sections? 
if this is just the tool, then dont worry about

Comment (hassount) [reply Source/Lib/Encoder/Codec/EbInitialRateControlProcess.c:202]: Gotcha, @1480c1 maybe something to revisit we implement the CI

**Pull request merged**